src/pages/docs/v3/modifiers/index.mdx

```diff
@@ -0,0 +1,83 @@
+---
+navigationLabel: Modifiers
+order: 4
+---
+
+# Modifiers
+
+A `modifier` is an object (middleware or plugin) that modifies the `x` and `y`
+coordinates in some way, or provides useful data to the consumer. Popper
+provides the most useful modifiers in its core already for you to import.
+Anything you don't use will be tree-shaken away.
+
+## Example
+
+A very basic modifier that shifts the coordinates by 1 pixel:
+
+```js
+const shiftBy1Px = {
+  // The name of the modifier.
+  name: 'shiftBy1Px',
+  // The main logic which returns new coordinates.
+  fn({ coords }) {
```

Comment:
Wonder if this should be `x` and `y` instead of `coords`, which matches other parts of the API. Small downside, can't `return coords`, rather `return {x, y}`

Reply:
I like x and y, it also makes it clearer the API is pure and side-effects free (you need to return a new object rather than a possibly mutated one.